setup.cfg

```diff
@@ -1,2 +1,39 @@
-[wheel]
-universal=1
+[metadata]
+name = incremental
+version = attr: incremental._setuptools_version
```

Comment:
This seems like a cleaner example to give if it works.  If taking this, remove the `def` added down below.

```suggestion
version = attr: incremental.__version__.public
```

Reply:
ModuleNotFoundError: No module named 'incremental.__version__'

Reply:
Welp, definitely a sign of a thing to fix as has been discussed on IRC.